src/basic/log.c

```diff
@@ -607,15 +632,19 @@ static int write_to_journal(
 
         log_do_header(header, sizeof(header), level, error, file, line, func, object_field, object, extra_field, extra);
 
-        struct iovec iovec[4] = {
+        struct iovec iovec[JOURNAL_MAX_ENTRIES] = {
```

Comment:
Does JOURNAL_MAX_ENTRIES need to be so big? `struct iovec` is 16 bytes, so we're allocating 16k here on the stack.

Reply:
I copied the max amount of entries that a single journal message can contain. We can certainly lower this though. I'll put in a lower value

Reply:
Lowered to 128. This gives us space for 64 fields